client/pages/live-map.jsx

```diff
@@ -95,17 +120,22 @@ class LiveMap extends React.Component {
           <Map
 
             google={this.props.google}
-                containerStyle={containerStyle}
+            containerStyle={containerStyle}
             initialCenter={this.state.mapCenter}
             zoom={10} >
 
             {this.state.vehicles.map(vehicle =>
-                  <Marker key={vehicle.vehicleId}
-                  position={{ lat: vehicle.coords.lat, lng: vehicle.coords.lng }}
-                  />
-            )}
 
-          </Map >
+              <Marker key={vehicle.vehicleId}
+                position={{ lat: vehicle.coords.lat, lng: vehicle.coords.lng }}
+                icon={{
+                  url: vehicle.stopped ? 'https://upload.wikimedia.org/wikipedia/commons/0/07/Button_Icon_Red.svg' : 'https://upload.wikimedia.org/wikipedia/commons/f/ff/Green_icon.svg',
```

Comment:
You should download these SVGs to your server/public directory so you don't have to worry about external links breaking.